cirq/google/calibration/phased_fsim.py

```diff
@@ -220,6 +220,22 @@ class PhasedFSimCalibrationRequest(abc.ABC):
     pairs: Tuple[Tuple[Qid, Qid], ...]
     gate: Gate  # Any gate which can be described by cirq.PhasedFSim
 
+    @staticmethod
+    def _create_pairs_from_moment(moment: Moment):
+        gate = None
+        pairs = []
+        for op in moment:
+            if gate is None:
+                gate = op.gate
+            elif gate != op.gate:
```

Comment:
Shouldn't that be gated against a check that operation is a GateOperation? Not all operations have gate?

Reply:
Changed to check if op.gate is not None, since it could be a TaggedOperation or some other class instead.